src/test/java/org/kohsuke/github/ReleaseTest.java

```diff
@@ -0,0 +1,67 @@
+package org.kohsuke.github;
+
+import org.junit.Test;
+
+import java.util.UUID;
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertThrows;
+
+public class ReleaseTest extends AbstractGitHubWireMockTest {
+
+    @Test
+    public void testCreateSimpleRelease() throws Exception {
+        GHRepository repo = gitHub.getRepository("hub4j-test-org/testCreateRelease");
+
+        String tagName = UUID.randomUUID().toString();
```

Comment:
Doh.  You can't do this. The tag name needs to be stable across multiple runs so that the recorded data can be used.  

Suggest you do a stable name and then use a `try-finally` to always do clean at the end of the test: 

```

String tagName = mockGitHub.getMethodName();
GHRelease release = repo.createRelease(tagName);
try {
    // test actions
} finally { 
    rel.delete();
}
```

Reply:
Ha thanks for the pointer, I was actually looking for that issue.

Reply:
Alright, should be fixed. Let's see what happens